ActionHeader action_header
uint8 status
uint8 PENDING         = 2
uint8 ACTIVE          = 3
uint8 PREEMPTED       = 4
uint8 SUCCEEDED       = 5
uint8 ABORTED         = 6
uint8 REJECTED        = 7
